x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/license/LicensingTests.java

```diff
@@ -235,8 +293,42 @@ private void enableLicensing(License.OperationMode operationMode) throws Excepti
             // re-apply the update in case any node received an updated cluster state that triggered the license state
             // to change
             for (XPackLicenseState licenseState : internalCluster().getInstances(XPackLicenseState.class)) {
-                licenseState.update(operationMode, true, null);
+                licenseState.update(operationMode, true, Long.MAX_VALUE, null);
             }
         }, 30L, TimeUnit.SECONDS);
     }
+
+    private void setLicensingExpirationDate(License.OperationMode operationMode, long expirationDate) throws Exception {
+        // do this in an await busy since there is a chance that the setting expiration date of the license is
+        // overwritten by some other cluster activity and the node throws an exception while we
+        // wait for things to stabilize!
+        assertBusy(() -> {
+            // first update the license so we can execute monitoring actions
```

Comment:
Do we have to do the double update dance here? `setLicensingExpirationDate` is the only thing that calls update with anything other than Long.MAX_VALUE, what cluster state could trigger the license state (Expiration date) to change ?

Reply:
Probably, not. Removed the second update, left hte assertBusy though.